scipy/sparse/csgraph/_flow.pyx

```diff
@@ -97,6 +110,9 @@ def maximum_flow(csgraph, source, sink):
     .. [2] Cormen, T. H. and Leiserson, C. E. and Rivest, R. L. and Stein C.
            Introduction to Algorithms. Second Edition. 2001. MIT Press.
     .. [3] https://en.wikipedia.org/wiki/Graph_cuts_in_computer_vision
+    .. [4] Dinic, E.A. (1970).
+           Algorithm for solution of a problem of maximal flow in a network with
+           power estimation.
```

Comment:
The reference is not complete here. At least it is missing the journal

Reply:
I copied the citation from [here](https://www.semanticscholar.org/paper/Algorithm-for-solution-of-a-problem-of-maximal-flow-Dinic/13e9613b5a06776c15f7adc562f6f8a385fc4f29). I will replace it with the one from Google scholar as it has all the details. Thanks for pointing out.